projects/plugins/jetpack/extensions/blocks/subscriptions/panel.scss

```diff
@@ -0,0 +1,11 @@
+.jetpack-subscribe-pre-publish-panel,
+.jetpack-subscribe-post-publish-panel {
+	.jetpack-inspector-notice {
+		margin: 0;
+	}
+
+	.jetpack-subscribe-reader-count {
+		text-decoration: underline;
```

Comment:
With the underline, I really expected that text to be a link. Maybe we could transform this into an ExternalLink that would link out to `https://wordpress.com/people/followers/yoursite.com`?

Maybe we could start importing `getRedirectUrl` from `@automattic/jetpack-components` and start using it here, like we do here for example:
https://github.com/Automattic/jetpack/blob/39fa657bfe0aaca179275331752fd99a74ac1c41/projects/plugins/jetpack/_inc/client/discussion/subscriptions.jsx#L80-L82

(to get the site URL, we can do something like this: https://github.com/Automattic/jetpack/blob/b91373ca0e69043d00d39f689750fa33ddb16083/projects/plugins/jetpack/extensions/blocks/contact-form/components/jetpack-newsletter-integration-settings-plugin-state.js#L79
)

Reply:
I'd actually experimented with that exact implementation prior to the rebase. Unfortunately, the followers page splits email and WPCOM followers into two separate tabs; the count we present within the subscribe block is actually the count of both email and WPCOM followers ([context](https://github.com/Automattic/jetpack/pull/22912#issuecomment-1043670290)).

We'll likely pursue this in the long term after tweaking the followers page in the future.